packages/headless/src/features/search/search-actions.ts

```diff
@@ -110,6 +114,12 @@ export const executeSearch = createAsyncThunk<
   'search/executeSearch',
   async (searchAction: TransitiveSearchAction, config) => {
     const state = config.getState();
+
+    if (
```

Comment:
Could you do something like this instead? 

https://github.com/coveo/ui-kit/pull/4129/files#diff-644461440e8a66297a77b95e54021c1e4840bad4c475302dea37bfc17945091cR70

Reply:
I will remove changes to this file entirely